docs/log-in-as-po.md

```diff
@@ -1,24 +1,55 @@
 # Log in as public officer
 
-1. Go to the [Docs portal](https://docs.developer.tech.gov.sg/) and click **Log in with TechPass**.
-<kbd>![log-in-with-techpass](assets/images/access-sgts-services-using-techpass/first.png)</kbd>
-2. Sign in to your TechPass account or choose your TechPass account.
-<kbd>![sign-in](assets/images/access-sgts-services-using-techpass/log-in-with-techpass.png)</kbd>
+Public officer can log in to TechPass using your non-SE GSIB device or your Government Managed Device (GMD).
+
+**To log in with a non-SE GSIB device**
+
+1. Go to the [Docs portal](https://docs.developer.tech.gov.sg/) and click **Log in**.
 
-If you are signing in from your GMD, you will be prompted to authenticate your WOG account.
+<kbd>![new-mfa](assets/images/onboarding/po-non-se/new-docs-portal.png)</kbd>
 
-3. Enter the verification code displayed for your SG Govt M365 profile on your Authenticator app.
+2. Click **Log in with TechPass**.
 
-<kbd>![verify-wog-for-po](assets/images/access-sgts-services-using-techpass/verification-code-po.png)-->
+<kbd>![new-mfa](assets/images/onboarding/po-non-se/new-docs.png)</kbd>
 
-You will now be prompted to approve your TechPass sign-in. A number will be shown on your browser.
+3. Sign in to your TechPass account or choose your TechPass account.
 
- <kbd>![number-mfa](assets/images/onboarding/po-non-se/mfa-number-displayed-on-screen.png)</kbd>
+<kbd>![sign-in](assets/images/access-sgts-services-using-techpass/log-in-with-techpass.png)</kbd>
+
+4. You will be prompted to authenticate your WOG account. A number will be shown on your browser.
 
-4. On the Authenticator app, enter the number shown, and tap **Yes** to authenticate this sign-in.
+<kbd>![new-mfa](assets/images/onboarding/po-non-se/new-mfa.png)</kbd>
 
 You are now successfully logged in with TechPass.
 
 <kbd>![after-po-login](assets/images/access-sgts-services-using-techpass/final.png)</kbd>
 
 ?> To log out of TechPass, click **Log Out** and confirm which account you would like to log out.
+
+**To log in with a GMD device**
+
+1. Go to the [Docs portal](https://docs.developer.tech.gov.sg/) and click **Log in**.
+
+<kbd>![new-mfa](assets/images/onboarding/po-non-se/new-docs-portal.png)</kbd>
+
+2. Click **Log in with TechPass**.
+
+<kbd>![new-mfa](assets/images/onboarding/po-non-se/new-docs.png)</kbd>
+
+3. Sign in to your TechPass account or choose your TechPass account.
+
+<kbd>![sign-in](assets/images/access-sgts-services-using-techpass/log-in-with-techpass.png)</kbd>
+
+4. You will be prompted to authenticate your WOG account. Enter the verification code displayed for your SG Govt M365 profile on your Authenticator app.
```

Comment:
there's 2 4s and the image is not showing up
<img width="1052" alt="image" src="https://user-images.githubusercontent.com/11743256/224989843-71d2ef09-6e10-44c9-a3fc-2809092aadcd.png">

Reply:
fixed. as mentioned, images worked fine on local preview. will fix this.